diagrams/ecosystemdetail0.svg

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" standalone="yes"?>
+<svg version="1.1" viewBox="0 0 960 720" fill="none" xmlns="http://www.w3.org/2000/svg">
+	<defs>
+		<marker id="arrow" viewBox="0 0 6 4" refX="1" refY="2" fill="#000"
+      markerWidth="6" markerHeight="4" orient="auto">
+    <path d="M0 0v4l6 -2z" />
+    </marker>
+		<symbol id="gone" viewBox="-40 -40 960 720">
+			<g transform="rotate(-45)">
+				<circle r="25" fill="red"/>
+				<circle r="15" fill="#fff"/>
+				<rect x="-15" y="-5" width="30" height="10" fill="red"/>
+				<rect class="none" x="-17" y="-4.5" width="34" height="9" fill="red"/>
+			</g>
+    </symbol>
+		<style><![CDATA[
+			path.link {stroke:#000;stroke-width:4;marker-end: url(#arrow)}
+			rect, circle {stroke:#000;stroke-width:1px}
+			rect.none {stroke:none}
+			text {fill:#000;font-size:14px;font-family:sans-serif}
+			text[aria-level ='1']{font-size:2em;font-weight:bold;font-style:italic}
+			text[aria-level = '2']{font-size:1.5em;font-weight:bold}
+			#background {fill:#fff;width:100%;height:100%;stroke:none}
+		]]></style>
+	</defs>
+<rect id="background"/>
```

Comment:
Is the filled background needed?

Similar comment for the other diagrams with solid vs transparent backgrounds.  Should they all be the same?  And I would guess transparent?  Or is there some dark background use case where they should all be filled white?

Reply:
If they are transparent, they may cause trouble for people using dark mode when browsing.

Reply:
+1 to @dlongley - I often get caught out because don't have the same background colour as the person who created an image, and it becomes unreadable.

Suggest we resolve this comment (and make sure *all* images have a solid background...)

Reply:
background added everywhere in bd4f6b3